src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Extensions/GraphQLTypeExtensions.cs

```diff
@@ -1,15 +1,20 @@
+using System;
+using System.Linq;
 using GraphQL.Types;
 
 namespace OrchardCore.ContentManagement.GraphQL;
 
-public static class FieldTypeExtensions
+public static class GraphQLTypeExtensions
```

Comment:
This is a breaking-change?

Reply:
When only used as extension methods, no. And I don't expect any caller outside of our own code.